resources/public/api/conf/1.0/scenarios/submitBusinessEOPS_test_scenarios.md

```diff
@@ -0,0 +1,71 @@
+<p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
+<table>
+    <thead>
+        <tr>
+            <th>Header Value (Gov-Test-Scenario)</th>
+            <th>Scenario</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><p>N/A - DEFAULT</p></td>
+            <td><p>Simulate success response.</p></td>
+        </tr>
+        <tr>
+            <td><p>MATCHING_RESOURCE_NOT_FOUND</p></td>
+            <td><p>Simulate scenario where no data found.</p></td>
+        </tr>
+        <tr>
+            <td><p>RULE_ALREADY_SUBMITTED</p></td>
+            <td><p>Simulate scenario where the user has previously submitted an End of Period Statement for this business' accounting period.</p></td>
+        </tr>
+        <tr>
+            <td><p>RULE_CLASS4_OVER_16</p></td>
+            <td><p>Simulate scenario where the user has requested a National Insurance Class 4 exemption but the individual’s age is greater than or equal to 16 years old on the 6th April of the current tax year.</p></td>
+        </tr>
+        <tr>
+            <td><p>RULE_CLASS4_PENSION_AGE</p></td>
+            <td><p>Simulate scenario where the user has requested a National Insurance Class 4 exemption but the individual's age is less than their State Pension age on the 6th April of the current tax year.</p></td>
+        </tr>
+        <tr>
+            <td><p>RULE_CONSOLIDATED_EXPENSES_C55316</p></td>
+            <td><p>Simulate scenario where self employment consolidated expenses are not allowed if the cumulative turnover amount exceeds the threshold.</p></td>
+        </tr>
+        <tr>
+            <td><p>RULE_CONSOLIDATED_EXPENSES_C55503</p></td>
+            <td><p>Simulate scenario where UK property consolidated expenses are not allowed if the cumulative turnover amount exceeds the threshold.</p></td>
+        </tr>
+        <tr>
+            <td><p>RULE_EARLY_SUBMISSION</p></td>
+            <td><p>Simulate scenario where the user has tried to make their End of Period Statement declaration before the accounting period has ended.</p></td>
+        </tr>
+        <tr>
+            <td><p>RULE_FHL_PRIVATE_USE_ADJUSTMENT</p></td>
+            <td><p>Simulate scenario for UK Furnished Holiday Lettings, the private use adjustment exceeds the total allowable expenses.</p></td>
+        </tr>
+        <tr>
+            <td><p>RULE_LATE_SUBMISSION</p></td>
+            <td><p>Simulate scenario where the user has tried to make their End of Period Statement declaration too late.</p></td>
+        </tr>
+        <tr>
+            <td><p>RULE_MISMATCHED_END_DATE_C55013</p></td>
```

Comment:
I don't think we want to be exposing error codes in these scenarios. can they be renamed?

Reply:
Updated so there are no references to BVR codes